tekton/README.md

```diff
@@ -17,144 +17,13 @@ The Pipelines and Tasks in this folder are used for:
 
 To start from scratch and use these Pipelines and Tasks:
 
-1. [Install Tekton v0.3.1](https://github.com/tektoncd/pipeline/blob/master/tekton/README.md#install-tekton)
+1. [Install Tekton](https://github.com/tektoncd/pipeline/blob/master/tekton/README.md#install-tekton)
 1. [Setup the Tasks and Pipelines](https://github.com/tektoncd/pipeline/blob/master/tekton/README.md#setup)
```

Comment:
@dibyom While reviewing PR found that link is not properly configured to point to exact path in the original code
as this is part of README can you please handle this also as part of this PR itself 
```suggestion
1. [Setup the Tasks and Pipelines](https://github.com/tektoncd/pipeline/blob/master/tekton/README.md#install-tasks-and-pipelines)
```

Reply:
done!